tmp/lanelet2_extension/src/check_row.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright 2015-2022 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOLINTBEGIN(readability-identifier-naming)
+
+#include <lanelet2_extension/projection/mgrs_projector.hpp>
+#include <lanelet2_extension/utility/query.hpp>
+#include <lanelet2_extension/utility/utilities.hpp>
+#include <range/v3/algorithm/move.hpp>
+#include <range/v3/core.hpp>
+#include <range/v3/functional/overload.hpp>
+#include <range/v3/utility/copy.hpp>
+#include <range/v3/view/filter.hpp>
+#include <range/v3/view/transform.hpp>
+#include <range/v3/view/unique.hpp>
+
+#include <lanelet2_core/LaneletMap.h>
+#include <lanelet2_io/Io.h>
+#include <lanelet2_routing/RoutingGraph.h>
+#include <lanelet2_traffic_rules/TrafficRulesFactory.h>
+
+#include <algorithm>
+#include <iostream>
+#include <memory>
+#include <set>
+#include <utility>
+#include <vector>
+
+int main(int argc, char ** argv)
+{
+  if (argc < 2) {
+    std::cout << "usage: ros2 run lanelet2_extension check_row <map_path>";
```

Comment:
If you agree to change the execution file name, you shouldn't forget to change here as well.
```suggestion
    std::cout << "usage: ros2 run lanelet2_extension check_right_of_way <map_path>";
```